cmd/defenseclaw-setup/main.go

```
@@ -78,6 +83,25 @@ func legacyGatewayAutoStartCommand(gatewayPath string) string {
 	return `"` + gatewayPath + `" start`
 }
 
+func runCommandUTF16Units(command string) int {
+	return len(utf16.Encode([]rune(command)))
+}
+
+func validateRunCommand(command string) error {
+	if strings.ContainsRune(command, '\x00') {
+		return errors.New("Windows Run command contains an embedded NUL")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)